### ISSUE TITLE
OpenSearchDescription points to localhost

### ISSUE THREAD

**Issue by IcecreamSlut** [mwmbl/mwmbl#2]
I don't know this is intentional at the moment, but notice this when I was poking around the page.

https://mwmbl.org/
Search for `application/opensearchdescription+xml` in the mark up. Which shows this:
![image](https://user-images.githubusercontent.com/51807808/147411275-6b102b4c-a780-44c6-bed7-24e2518a4a15.png)

https://search.brave.com/
```
<link rel="search" type="application/opensearchdescription+xml" title="Brave Search" href="https://cdn.search.brave.com/serp/v1/static/brand/c57da39655b0b08603d88711f8e33aae50500cbcd8d2fc70a0d01e105cbd0985-opensearch.xml">
```

**Comment by daoudclarke** [comment]
Oops, good spot. Needs fixing.

**Comment by daoudclarke** [comment]
I originally implemented OpenSearch but I think I broke it completely at some point, so in addition to @IcecreamSlut's observation above, we need to fix all the issues and test it end-to-end.

**Comment by daoudclarke** [comment]
Yes, you're right. My plan is to implement some other key features first that will make Mwmbl more usable. Right now it is more of a toy.

**Comment by JulienPalard** [comment]
Thanks @AdJaGu, it works for me, I was able to add it to Firefox too!

**Comment by JulienPalard** [comment]
Thanks!
